php-nginx/Dockerfile

```diff
@@ -16,9 +16,6 @@
 
 FROM gcr.io/google_appengine/debian8
 
-# Cache busting for security update
-ENV UPDATED_AT 2016-11-15
```

Comment:
Or perhaps, we can swap the environment variable setting with the apt-get updates so that every time we build a new php version, we get the latest security updates.